x/dsmr/block.go

```diff
@@ -16,6 +18,8 @@ import (
 
 const InitialChunkSize = 250 * 1024
 
+var ErrFailedSigVerification = errors.New("failed to verify signature")
```

Comment:
Can we make this more detailed ie. at least include the signature type and the type of container the signature was over like `"failed to verify bls chunk signature"`